### ISSUE TITLE
[🚀 Feature]: Selenium Manager use binary from Browser Options

### ISSUE THREAD

**Issue by titusfortner** [SeleniumHQ/selenium#11351]
### Feature and motivation

Evaluate the browser version at a specific location in order to know which driver to get. I think it would make sense to support this with `--browser` instead of creating a new flag. 

From a bindings implementation standpoint we'll want to inspect the `binary` key in the `goog:chromeOptions` capability to get this and pass it to the selenium-manager binary.

### Usage example

```
selenium-manager --browser "/Applications/Google Chrome Dev.app/Contents/MacOS/"
```
and
```
selenium-manager --browser "/Applications/Google Chrome Dev.app/Contents/MacOS/Google Chrome Dev"
```

**Comment by bonigarcia** [comment]
I would not reuse the flag `--browser` for the browser path, since `--browser` is supposed to be the browser name (`chrome`, `edge`, etc.). Otherwise you need to infer the browser name from that path. I would create another flag, such as `--browser-path`.

**Comment by titusfortner** [comment]
You're right. Keep things more obvious

**Comment by bonigarcia** [comment]
This is already implemented in the Rust side (#11496). Now the bindings can start using it. 

Is `C-java` supposed to be also in the labels?

**Comment by diemol** [comment]
Implemented in all bindings. Closing.
